src/main/java/domain/Crew.java

```diff
@@ -0,0 +1,95 @@
+package domain;
+
+import static domain.AbsencePenalty.LATE_TO_ABSENCE_THRESHOLD;
+import static domain.AttendanceResult.ABSENCE;
+import static domain.AttendanceResult.LATE;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import util.Convertor;
+
+public class Crew implements Comparable<Crew> {
+    private final String userName;
+    private final AttendanceHistories attendanceHistories;
+
+    public Crew(String userName, List<LocalDateTime> histories, LocalDate standard) {
+        validateName(userName);
+        this.userName = userName;
+        this.attendanceHistories = new AttendanceHistories(histories, standard);
+    }
+
+    public void validateName(String userName) {
+        if (userName.length() >= 5) {
```

Comment:
5라는 숫자는 다른 사람 입장에서는 한눈에 이해하기 어려운 숫자인 것 같아요. 👀
적절히 상수로 추출하여 의도를 드러내보면 어떨까요? 😃

추가적으로  이름이 네 글자 이하여야한다고 했으면,  5보다는 4가 조금 더 의도를 드러내기가 쉬운 숫자일 것 같아요. 
```suggestion
        if (userName.length() > 4) {
```

Reply:
앗 놓친 부분이네요! 로키 말대로 상수로 추출하여 의도를 정확히 드러내는 것이 좋아보입니다!